recipes/pycraf/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --offline --single-version-externally-managed --record record.txt
```

Comment:
Please remove the build scripts and use the `script` field with the pip invocation

```yaml
script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

like in the example.